src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -76,101 +80,112 @@ public enum PropertyType {
 		 */
 		HAVE
 	}
-	
-	@SuppressWarnings("null")
+
 	private Expression<? extends T> expr;
-	
+
 	/**
+	 * Registers a property condition. Use [!PATTERN] to set optional patterns for the negated part of the syntax.
+	 * Example <code>[!no|a] [direct] line of sight</code> this means if <code>no</code> part was used it will mark the pattern as negated.
```

Comment:
Since you updated the description to correctly give the example, this should be updated too
```suggestion
	 * Example <code>([!no]|a] [direct] line of sight</code> this means if <code>no</code> part was used it will mark the pattern as negated.
```